data/pathfinder/paizo/player_companion/blood_of_fiends/bof_abilities_race.lst

```diff
@@ -75,6 +75,8 @@ Vile Kiss			KEY:QlippothSpawn ~ Vile Kiss			CATEGORY:Special Ability	TYPE:Trait.
 Bent Body			KEY:RakshasaSpawn ~ Bent Body			CATEGORY:Special Ability	TYPE:Trait.RaceTrait.TieflingRaceTrait.RakshasaSpawnRaceTrait	PREMULT:1,[PREABILITY:1,CATEGORY=Special Ability,RakshasaSpawn ~ Bent Body],[!PREABILITY:1,CATEGORY=Special Ability,TYPE.RaceTrait]	PREMULT:1,[PRETEMPLATE:1,Tiefling (Rakshasa-Spawn)],[PREABILITY:1,CATEGORY=Special Ability,Tiefling ~ Adoptive Race Rakshasa-Spawn]		DESC:While nothing is actually reversed, your bones and limbs are slightly out of alignment. You gain a +2 trait bonus to CMD when resisting grapple attempts.																																		SOURCEPAGE:p.23
 Tough Skin			KEY:RakshasaSpawn ~ Tough Skin		CATEGORY:Special Ability	TYPE:Trait.RaceTrait.TieflingRaceTrait.RakshasaSpawnRaceTrait	PREMULT:1,[PREABILITY:1,CATEGORY=Special Ability,RakshasaSpawn ~ Tough Skin],[!PREABILITY:1,CATEGORY=Special Ability,TYPE.RaceTrait]	PREMULT:1,[PRETEMPLATE:1,Tiefling (Rakshasa-Spawn)],[PREABILITY:1,CATEGORY=Special Ability,Tiefling ~ Adoptive Race Rakshasa-Spawn]	DESC:You retain a small amount of your ancestors' unyielding skin. You gain a +1 trait bonus to AC when opponents attempt to confirm critical hits with either bludgeoning or slashing weapons.																													SOURCEPAGE:p.23
 
+Variant Ability								KEY:Tiefling ~ Variant Ability		CATEGORY:Special Ability	TYPE:SpecialQuality.TieflingRacialTrait.TieflingSLA.SpellLike	PREMULT:1,[PREABILITY:1,CATEGORY=Special Ability,Tiefling ~ Variant Ability],[!PREABILITY:1,CATEGORY=Special Ability,TYPE.TieflingSLA]																														DESC:Some Tieflings are blessed or cursed with unusual abilities.																													BONUS:ABILITYPOOL|Tiefling Variant Ability|1																																			COST:0	SOURCEPAGE:0
```

Comment:
SOURCEPAGE should not be blank or 0. Please cite the correct page, which appears to be 23 or 24 based upon the preceding ability.